.github/workflows/create_client_artifacts.yml

```diff
@@ -0,0 +1,65 @@
+name: Create Client Artifacts
```

Comment:
This approach leaves open the potential for the client files to not match the client files used for the compiled binaries and tests. Specifically, in the case that the *.mako files change but the updated *.proto and similar are not checked in afterwards. That case could result in the main build (which will regenerate the files) using different versions of the files than what is in the repo.

Ideally we that would never happen since we would catch it in code reviews, but making it fool-proof couldn't hurt.